pallets/randomness/src/migrations.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-2022 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::Config;
+use frame_support::{
+	pallet_prelude::PhantomData,
+	storage::migration::*,
+	traits::{Get, OnRuntimeUpgrade},
+	weights::Weight,
+};
+
+/// Removes `NotFirstBlock` storage item from storage and clears it
+pub struct RemoveNotFirstBlock<T>(PhantomData<T>);
+impl<T: Config> OnRuntimeUpgrade for RemoveNotFirstBlock<T> {
+	fn on_runtime_upgrade() -> Weight {
```

Comment:
We can just `kill_storage` or similar

Reply:
We already need code changes (and runtime upgrade) to remove the code that uses this storage item. So might as well clear the storage item in the same runtime upgrade.

In theory, we could even use a hotfix extrinsic, but I don't see the point when we know that we need a runtime upgrade regardless.